packages/medulas-react-components/src/pages/PageColumn/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { makeStyles, Theme } from '@material-ui/core';
+import React from 'react';
+import Form, { useForm } from '../../components/forms/Form';
+import Block from '../../components/Block';
+import Button from '../../components/Button';
+import Hairline from '../../components/Hairline';
+import Img from '../../components/Image';
+import logo from '../assets/logo.svg';
+import logoBlack from '../assets/logoBlack.svg';
+import EmptyHeader from './EmptyHeader';
+import SubtitleSection from './SubtitleSection';
+import TitleSection from './TitleSection';
+import useTheme from '@material-ui/styles/useTheme';
+import { FormApi } from 'final-form';
+
+interface Props {
+  readonly leftMenu: () => JSX.Element;
+
+  readonly icon: 'white' | 'black';
+  readonly nextMsg: string;
+  readonly onSubmit: (values: object) => void;
+  readonly formRender: (form: FormApi) => JSX.Element;
+  readonly validation?: (values: object) => object | Promise<object>;
+
+  readonly primaryTitle: string;
+  readonly secondaryTitle: string;
+  readonly subtitle: string;
+
+  readonly renderHeader?: () => JSX.Element;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  logo: {
+    bottom: '80px',
+    position: 'relative',
+    display: 'flex',
+    margin: '0 auto',
+  },
+}));
+
+const Layout = ({
+  formRender,
+  onSubmit,
+  icon,
+  primaryTitle,
+  secondaryTitle,
+  subtitle,
+  nextMsg,
+  renderHeader,
+  leftMenu,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+  const theme = useTheme<Theme>();
+
+  const { form, handleSubmit, submitting, invalid, validating } = useForm({
+    onSubmit,
+  });
+
+  return (
+    <Block
```

Comment:
This component is complex, I believe we can simplify it. Some hints:
1. The login page is the only view which has an image on the left, so we can remove the `leftMenu()` and code hre the content.
2. Material ui has a [container](https://material-ui.com/components/container/)  component which can work for the content, you can specify margins automatically.

If you do not have time and you are blocked, just code the first one and create a new ticket for the second one.